frontend/src/components/ItemMetadataForm.js

```diff
@@ -30,18 +49,99 @@ const ItemMetadataForm = ({
     helperText={ errors.description ? errors.description.join(' ') : null }
     disabled={ disabled }
     label='Description'
+    placeholder='Enter a description (markdown can be used)'
     margin='normal'
     multiline
     onChange={ event => onChange && onChange({ description: event.target.value }) }
     rows={ 4 }
     value={ description }
+    InputLabelProps={ { shrink: true } }
+  />
+
+  <FormControlLabel
+    checked={downloadable}
+    onChange={ event => onChange && onChange({ downloadable: event.target.checked }) }
+    control={
+      <Checkbox
+      />
+    }
+    label="Allow media to be downloaded and indexed by search engines"
   />
+
+  <Grid container spacing={8}>
+    <Grid item xs={12} sm={6}>
+      <TextField
+        className={classes.publishedAt}
+        fullWidth
+        error={ !!errors.publishedAt }
+        helperText={ errors.publishedAt ? errors.publishedAt.join(' ') : null }
+        value={publishedAt ? moment(publishedAt).format("YYYY-MM-DD") : ''}
+        label="When the item will be published"
```

Comment:
``label="Publication date"``

Reply:
I think this needs to support setting the publication time as well as date **OR** we can just remove the field from the form.